src/HE.FMS.Middleware.Claims/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
do we need gitignore per function project? I believe it should be sufficient to have just single one in src directory

Reply:
nice catch. Removed